templates/dump-data.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+
+set -e
+
+DUMP_CMD="mysqldump --no-create-info --complete-insert  --skip-triggers --no-create-db --host {{ OLD_DALITE_MYSQL_HOST }} --user {{ OLD_DALITE_MYSQL_USER }} --password='{{ OLD_DALITE_MYSQL_PASSWORD }}' {{ OLD_DALITE_MYSQL_NAME }}"
```

Comment:
I think we should quote the host, username and password here in case they include any spaces or special shell characters such as `&`